encord/objects/coordinates.py

```diff
@@ -311,7 +312,8 @@ class SkeletonCoordinate(BaseDTO):
     visibility: Optional[Visibility] = None
 
 
-class SkeletonCoordinates(BaseDTO):
+@dataclass(frozen=True)
```

Comment:
Why migrate from BaseDTO to dataclass?
At the time, it was flagged by Sergei as easier if we don't write our own serialisation code.